Panic when using `tctl` with remote auth server

### Description

Attempted to run a `tctl` command against a remote auth server. My profile `~/.tsh` directory is empty and I did not specify an identity to use on the command line.

**What happened**:

```
$ ../6.2/tctl --auth-server=proxy.example.com users add --logins=trent  beaker --debug
DEBU             Debug logging has been enabled. common/tctl.go:322
DEBU             No config file or identity file, loading auth config via extension. common/tctl.go:348
panic: runtime error: index out of range [0] with length 0
goroutine 1 [running, locked to thread]:
github.com/gravitational/teleport/tool/tctl/common.loadConfigFromProfile(0xc0002a6780, 0xc000341000, 0xc0009bfc60, 0x1, 0x1)
	/go/src/github.com/gravitational/teleport/tool/tctl/common/tctl.go:426 +0xf56
github.com/gravitational/teleport/tool/tctl/common.applyConfig(0xc0002a6780, 0xc000341000, 0x7, 0x6, 0xc000052284)
	/go/src/github.com/gravitational/teleport/tool/tctl/common/tctl.go:349 +0xb3d
github.com/gravitational/teleport/tool/tctl/common.Run(0xc0009bfed8, 0xa, 0xa)
	/go/src/github.com/gravitational/teleport/tool/tctl/common/tctl.go:147 +0xbb3
main.main()
	/go/src/github.com/gravitational/teleport/tool/tctl/main.go:36 +0x27e
```

**What you expected to happen**:

An appropriate error message and exit.

### Reproduction Steps

As minimally and precisely as possible, describe step-by-step how to reproduce the problem.
1. Ensure your `~/.tsh` directory is empty
2. Run `tctl --auth-server $REMOTE_AUTH_SERVER_ADDR $SOME_TCTL COMMAND`

--- a/tool/tctl/common/tctl.go
+++ b/tool/tctl/common/tctl.go
@@ -423,7 +423,7 @@ func loadConfigFromProfile(ccf *GlobalCLIFlags, cfg *service.Config) (*AuthServi
 
 	proxyAddr := ""
 	if len(ccf.AuthServerAddr) != 0 {
-		proxyAddr = cfg.AuthServers[0].Addr
+		proxyAddr = ccf.AuthServerAddr[0]
 	}
 
 	profile, _, err := client.Status("", proxyAddr)

